gamemodes/terrortown/gamemode/shared/lang/en.lua

```diff
@@ -1119,19 +1119,19 @@ L.submenu_appearance_interface_title = "Interface"
 L.submenu_appearance_miscellaneous_title = "Misellaneous"
 
 L.submenu_gameplay_general_title = "General"
-L.submenu_gameplay_avoidroles_title = "Avoid Role Selection"
+L.submenu_gameplay_roles_title = "Avoid role selection"
```

Comment:
Headings are always with capital letters for every word. Same holds true for all of your changes.

Reply:
Additionally you can't just change the string identifier without changing it in the code as well.

Reply:
I forgot about this rule, yeah. For string 1218 it doesn't really matter because of this:
![image](https://user-images.githubusercontent.com/23385240/104123571-0112b800-535d-11eb-8a44-0917d8550b23.png)


> Additionally you can't just change the string identifier without changing it in the code as well.

Wait, how did I... I haven't change it...